src/diffusers/models/unet_2d_condition.py

```diff
@@ -104,6 +104,10 @@ class conditioning with `class_embed_type` equal to `None`.
             The dimension of `cond_proj` layer in timestep embedding.
         conv_in_kernel (`int`, *optional*, default to `3`): The kernel size of `conv_in` layer.
         conv_out_kernel (`int`, *optional*, default to `3`): The kernel size of `conv_out` layer.
+        patch_size (`int`, *optional*, default to `1`):
```

Comment:
Do we have text-conditioned checkpoints for Patch Diffusion? I'm not too keen on adding too many if-else statements if we don't have trained checkpoints for stable -diffusion

Reply:
Unfortunately, Patch Diffusion did not train any text-conditioned models to my knowledge, instead using dataset-specific class category embeddings. Ping @ericl122333 for more context.